android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/PolicyActivity.java

```diff
@@ -0,0 +1,75 @@
+package com.liferay.mobile.formsscreenletdemo.view;
+
+import android.graphics.Color;
+import android.support.design.widget.Snackbar;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.Button;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.liferay.mobile.formsscreenletdemo.R;
+import com.liferay.mobile.screens.asset.AssetEntry;
+import com.liferay.mobile.screens.asset.display.AssetDisplayInnerScreenletListener;
+import com.liferay.mobile.screens.asset.display.AssetDisplayListener;
+import com.liferay.mobile.screens.asset.display.AssetDisplayScreenlet;
+import com.liferay.mobile.screens.base.BaseScreenlet;
+import com.liferay.mobile.screens.util.AndroidUtil;
+
+/**
+ * @author Vitória Mendes
+ * @author Luísa Lima
+ */
+public class PolicyActivity extends AppCompatActivity implements AssetDisplayListener, AssetDisplayInnerScreenletListener {
+
+    private AssetDisplayScreenlet assetDisplayScreenlet;
+    private Toolbar toolbar;
+    private final String ENTRY_ID = "entryId";
```

Comment:
ENTRY_ID appears that it's the entryId value, it's better rename to ENTRY_ID_PARAM or something similar.

Reply:
You can solve this in two ways. You can put it on a Constants File that you only use for key values, or instead use ENTRY_ID_KEY

Reply:
okay, I'm agree.